src/views/open-api-docs-view/components/operation-sections/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react'
+import s from './operation-sections.module.css'
+
+/**
+ * Lays out columns for the examples and details of an operation.
+ *
+ * On large viewports, the exampleSlot is sticky and scrolls alongside
+ * the detailsSlot. On smaller viewports, the examplesSlot stacks above
+ * the detailsSlot. On the smallest viewports, it may be hidden entirely.
+ *
+ * TODO: determine breakpoint at which to hide examplesSlot, and implement.
+ */
+export function OperationSections({
```

Comment:
Basic layout component so that we can split out implementation of the "examples" and "details" columns.